dev-support/Jenkinsfile_GitHub

```diff
@@ -47,122 +49,157 @@ pipeline {
     }
 
     stages {
-        stage ('install yetus') {
+        stage ('scm checkouts') {
             steps {
+                echo "scm checkout WORKSPACE: ${WORKSPACE}"
+                dir("${WORKSPACE}/${SOURCEDIR}") {
+                    checkout scm
+                }
                 dir("${WORKSPACE}/${YETUS}") {
                     checkout([
-                        $class: 'GitSCM',
-                        branches: [[name: "${env.YETUS_VERSION}"]],
-                        userRemoteConfigs: [[ url: 'https://github.com/apache/yetus.git']]]
+                      $class: 'GitSCM',
+                      branches: [[name: "${env.YETUS_VERSION}"]],
```

Comment:
ugh. how have I never noticed this in the github handling.

I'll file a follow-on to stop doing this. we should be consuming the artifacts of other projects not hooking into their git repos.

Reply:
I'd like to extract the logic for grabbing either a released tarball or building a specified ref into a script that can be shared between the jobs.